ui/src/components/layout/DateAgo.vue

```diff
@@ -1,19 +1,11 @@
 <template>
-    <el-tooltip
```

Comment:
Why did you remove the tooltip here? This component is used in many different places, so just removing things is not okay.